Sources/Internal/VisibleItemsProvider.swift

```diff
@@ -951,9 +952,7 @@ final class VisibleItemsProvider {
     currentMonth = month
     currentMonthFrame = monthFrame
     while
-      bounds.contains(
-        CGPoint(x: currentMonthFrame.maxX - 1, y: currentMonthFrame.maxY - 1)
-          .alignedToPixels(forScreenWithScale: scale)),
```

Comment:
🤮 turns out all I needed was `CGRect.intersects`. I was doing this before because `CGRect(0, 0, 1, 1)` does not contain `CGPoint(1, 1)`. The rect intersects check handles this perfectly though, with no gross origin offsetting.